---
title: LOCALDB_ERROR_INTERNAL_ERROR | Documentos de Microsoft
ms.custom: ''
ms.date: 03/04/2017
ms.prod: sql
ms.reviewer: ''
ms.suite: sql
ms.technology: performance
ms.tgt_pltfrm: ''
ms.topic: reference
ms.assetid: 60c7a47b-114c-441b-869e-9d8ebfe4148e
caps.latest.revision: 8
author: stevestein
ms.author: sstein
manager: craigg
ms.openlocfilehash: 401b401051d1d2d943008ceb2bb537e27e2627bc
ms.sourcegitcommit: ee661730fb695774b9c483c3dd0a6c314e17ddf8
ms.translationtype: MT
ms.contentlocale: es-ES
ms.lasthandoff: 05/19/2018
---
# <a name="localdberrorinternalerror"></a>LOCALDB_ERROR_INTERNAL_ERROR
[!INCLUDE[appliesto-ss-xxxx-xxxx-xxx-md](../../includes/appliesto-ss-xxxx-xxxx-xxx-md.md)]
    
## <a name="details"></a>Detalles  
  
|||  
|-|-|  
|Nombre del producto|SQL Server|  
|Identificador del evento|264|  
|Origen del evento|SQL Server Local Database Runtime 12.0|  
|Componente|API de Local Database Runtime|  
|Texto del mensaje|Error inesperado en una llamada al método de API de la instancia de Local Database. Vea el registro de aplicación de Windows para obtener los detalles del error.|  
  
## <a name="explanation"></a>Explicación  
 Error inesperado.  
  
## <a name="user-action"></a>Acción del usuario  
 Vea el registro de eventos para obtener detalles.  
  
  
